### DISCUSSION TITLE
Sequoia image built on Sequoia host errors when run on Sonoma host

### DISCUSSION THREAD

**Discussion by andrewhamon** [cirruslabs/tart#961] [General]
I have built a Sequoia image from scratch on a Sequoia host. When i try to clone it on a Sonoma host, it fails with:

```
Error: dataCorrupted(Swift.DecodingError.Context(codingPath: [CodingKeys(stringValue: "hardwareModel", intValue: nil)], debugDescription: "", underlyingError: nil))
```

I've read https://github.com/cirruslabs/tart/issues/883#issuecomment-2273236519 and related issues, but I can't seem to find a straight answer: is there a way to run a Sequoia image, built on Sequoia, on a Sonoma host?

It might be worth making a sticky issue explaining what exactly the current limitations are when mixing and matching OS versions.

Sequoia version: 15.1.1
Sonoma version: 14.6.1

**Comment by fkorotkov** [comment]
Hi @andrewhamon, unfortunately that's the case. We have to use the same `hardwareModel` for booting which was used for creating of a particular virtual machine. It's unfortunate it's not backward compatible (improving error message in #962).

Sequoia on Sequoia introduced Apple ID login support. For security reasons, Apple made this support to use hardware encryption to work only on a particular host/guest pair. If you move a guest VM to another host, Apple ID will stop working. Our guess this broke running such images on Sonoma.
